components/automate-chef-io/content/docs/quickstart.md

```diff
@@ -62,37 +104,17 @@ Now that you have the `Vagrantfile` in your current directory, provision a new V
 vagrant up
 ```
 
-If this is your first time provisioning a VM, this operation may take a few
-minutes to download the base OS image.
+If this is your first time provisioning a VM, this operation may take a few minutes to download the base OS image.
 
-## Access the VM and Install Chef Automate
-
-1. Open a terminal session on your VM:
-
-     ```bash
-     vagrant ssh
-     ```
-
-1. Download and unzip the command line tool:
-
-     ```bash
-     curl https://packages.chef.io/files/current/automate/latest/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
-     ```
-
-1. Run
-
-     ```bash
-     sudo ./chef-automate deploy
-     ```
+When the command completes, you can access the web UI by browsing to `https://chef-automate.test/` (assuming you followed the above instructions and created an entry in `/etc/hosts`).
 
-    You will be prompted to accept our [Terms of Service][] and our [Master License and Services Agreement][].
+Your login credentials output to the terminal at the end of the installation process. 
+You can also find your login credentials in the `automate-credentials.toml` in your Chef Automate installation.  To access the `automate-credentials.toml`, first enter `vagrant ssh` in the command line, which puts you inside of your Chef Automate installation, and then enter `cat automate-credentials.toml`. Type `exit` to leave your Chef Automate installation and return to your main terminal environment. Chef Automate will continue running.
 
-Once you've accepted these agreements, the installer downloads, configures, and deploys Chef Automate.
-When the command completes, you can access the web UI by browsing to `https://chef-automate.test/` (assuming you followed the above instructions and created an entry in `/etc/hosts`).
-You can find login credentials by opening the `automate-credentials.toml`, which the installer has written in your current working directory.
+By installing Chef Automate with the Vagrantfile provided above you're automatically giving your consent to
```

Comment:
This section should move closer to where you're actually running `vagrant up`. I know that the code makes them accept it, but otherwise if they're just "yessing" through the prompts they might not realize what they're clicking. This makes it much more clear.

This currently reads to me like "now that you've run through all those steps, be aware that you agreed to all of this"